src/config.ts

```diff
@@ -86,6 +86,12 @@ export interface QueryConfig {
   preferredNominalAxis?: Channel;
   preferredFacet?: Channel;
 
+  // Not sure where these belong..
```

Comment:
They are not constraints, so definitely not here.  

They are post enumeration modification, which happens after CONSTRAINTS.  
Put it below all constraints  and add `// POST ENUMERATION MODIFICATION`